tests/NHSD.BuyingCatalogue.Solutions.Persistence.DatabaseTests/SolutionCapabilityRepositoryTests.cs

```diff
@@ -44,95 +44,39 @@ await SolutionEntityBuilder.Create()
                 await _solutionCapabilityRepository.ListSolutionCapabilities(Solution1Id, CancellationToken.None)
                     .ConfigureAwait(false);
 
             var solutionCapability = solutionCapabilityRequest.Should().ContainSingle().Subject;
-            solutionCapability.CapabilityId.Should().Be(_cap1Id);
-            solutionCapability.CapabilityName.Should().Be(cap1Name);
-            solutionCapability.CapabilityDescription.Should().Be(cap1Desc);
+            solutionCapability.CapabilityId.Should().Be(capDetails[0].id);
+            solutionCapability.CapabilityName.Should().Be(capDetails[0].name);
+            solutionCapability.CapabilityDescription.Should().Be(capDetails[0].desc);
         }
 
         [Test]
-        public async Task ShouldHaveMultipleCapabilities()
+        public async Task ShouldHaveMultipleCapabilitiesAsync()
         {
-            const string cap1Name = "Cap1";
-            const string cap1Desc = "Desc1";
-
-            const string cap2Name = "Cap2";
-            const string cap2Desc = "Desc2";
-
-            const string cap3Name = "Cap3";
-            const string cap3Desc = "Desc3";
-
-            await CapabilityEntityBuilder.Create()
-                .WithId(_cap1Id)
-                .WithName(cap1Name)
-                .WithDescription(cap1Desc)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
-
-            await CapabilityEntityBuilder.Create()
-                .WithId(_cap2Id)
-                .WithName(cap2Name)
-                .WithDescription(cap2Desc)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
-
-            await CapabilityEntityBuilder.Create()
-                .WithId(_cap3Id)
-                .WithName(cap3Name)
-                .WithDescription(cap3Desc)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
-
-            await SolutionCapabilityEntityBuilder.Create()
-                .WithCapabilityId(_cap1Id)
-                .WithSolutionId(Solution1Id)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
-
-            await SolutionCapabilityEntityBuilder.Create()
-                .WithCapabilityId(_cap2Id)
-                .WithSolutionId(Solution1Id)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
-
-            await SolutionCapabilityEntityBuilder.Create()
-                .WithCapabilityId(_cap3Id)
-                .WithSolutionId(Solution1Id)
-                .Build()
-                .InsertAsync()
-                .ConfigureAwait(false);
+            foreach (var capability in capDetails)
+            {
+                 await InsertCapabilityAsync(capability).ConfigureAwait(false);
+                 await InsertSolutionCapabilityAsync(capability.id, Solution1Id).ConfigureAwait(false);
```

Comment:
Inserting something for a solution, seems like the solution ID should be first in the parameter list.